docs/Concepts/4.md

```diff
@@ -1,60 +1,104 @@
 
-### Spam protection
+**Pros:**
 
-This property indicates that no adversary can flood the system (i.e., publishing a large number of messages in a short amount of time), either accidentally or deliberately, with any kind of message i.e. even if the message content is valid or useful.
-Spam protection is partly provided in `WAKU2-RELAY` through the [scoring mechanism](https://github.com/libp2p/specs/blob/master/pubsub/gossipsub/gossipsub-v1.1.md#spam-protection-measures) provided for by GossipSub v1.1.
-At a high level, peers utilize a scoring function to locally score the behavior of their connections and remove peers with a low score.
+- Low latency
+- Low resource requirements
 
-There's also further research being done in this domain, including the designing of (RLN (Rate Limiting Nullifiers))[./7.md]
+**Cons:**
 
-### Data confidentiality, Integrity, and Authenticity
+- Prone to censorship: node IPs can be blocked
+- Limited: Static number of nodes
+- Poor maintainability: Code needs to be changed to update the list
 
-Confidentiality can be addressed through data encryption whereas integrity and authenticity are achievable through digital signatures.
-These features are provided for in [WAKU-MESSAGE (version 1)](https://rfc.vac.dev/spec/14#version-1) and [WAKU-NOISE](https://rfc.vac.dev/spec/35/) through payload encryption as well as encrypted signatures.
+## Use your own nodes
 
-> `WAKU-NOISE` enables secure channel negotiation over Waku.
+Developers have the choice to run their own [nwaku](https://github.com/status-im/nim-waku/) nodes. Read [Nwaku Service Node](https://github.com/status-im/nwaku/tree/master/docs/operators) to learn how to run your own node.
 
-## Security Considerations
+Once you have started your nwaku node(s), you could decide to pass a list of predefined nodes as a way to bootstrap your own node.
 
-**Lack of anonymity/unlinkability in the protocols involving direct connections including [`13/WAKU2-STORE`](https://rfc.vac.dev/spec/13/) and [`12/WAKU2-FILTER`](https://rfc.vac.dev/spec/12/) protocols**:
-The anonymity/unlinkability is not guaranteed in the protocols like `13/WAKU2-STORE` and `12/WAKU2-FILTER` where peers need to have direct connections to benefit from the designated service.
-This is because during the direct connections peers utilize `PeerID` to identify each other,
-therefore the service obtained in the protocol is linkable to the beneficiary's `PeerID` (which counts as PII).
-For `13/WAKU2-STORE`, the queried node would be able to link the querying node's `PeerID` to its queried topics.
-Likewise, in the `12/WAKU2-FILTER`, a full node can link the light node's `PeerID`s to its content filter.
+**Pros:**
+
+- Low latency
+- Low resource requirements
+
+**Cons:**
+
+- Prone to censorship: node IPs can be blocked
+- Limited: Static number of nodes
+- Poor maintainability: Code needs to be changed to update the list
```

Comment:
ah, I see what was there. I'd just remove this section and focus on peer discovery mechanisms.